hkmc2/shared/src/test/mlscript/ucs/syntax/Else.mls

```diff
@@ -119,3 +119,22 @@ else
 //│ ║  l.116: 	    5 else 1
 //│ ╙──       	      ^^^^
 //│ = 8
+
+data class Box[A](value: A)
+
+:w
+fun foo(x) = if x is
+  Box(0) then "The box contains zero."
+  Box(1) then "The box contains one."
+  else "I can't tell you what's inside the box."
+  else "I don't want to tell you what's inside the box."
+  else "I shouldn't tell you what's inside the box."
+//│ ╔══[WARNING] This else clause makes the following branches unreachable.
+//│ ║  l.129: 	  else "I can't tell you what's inside the box."
```

Comment:
BTW now there are locations attached to keywords, so we should be able to report a more appropriate error location here.

Reply:
Resolved in b80d4bcd3b231729e084ce4cda1fb579608191cc.